patches/components-payments-content-BUILD.gn.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/components/payments/content/BUILD.gn b/components/payments/content/BUILD.gn
+index fa171bba474a1b6ab3ff7f968959a0ffd3ef6f3c..28d42c4e024d3646cad840d6520fef58f671d7bb 100644
+--- a/components/payments/content/BUILD.gn
++++ b/components/payments/content/BUILD.gn
+@@ -3,6 +3,7 @@
+ # found in the LICENSE file.
+ 
+ import("//build/config/chromeos/ui_mode.gni")
++import("//brave/components/payments/content/brave_payment_sources.gni")
```

Comment:
this entire patch can be done on one line using source += and deps +=

Reply:
resolved